sdk/servicebus/Azure.Messaging.ServiceBus/tests/SenderLiveTests.cs

```diff
@@ -88,6 +90,71 @@ await using (var scope = await ServiceBusScope.CreateWithTopic(enablePartitionin
             }
         }
 
+        [Test]
+        public async Task SenderCanSendAMessageBatch()
+        {
+            await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false))
+            {
+                await using var client = new ServiceBusClient(TestEnvironment.ServiceBusConnectionString);
+                ServiceBusSender sender = client.GetSender(scope.QueueName);
+                using ServiceBusMessageBatch batch = await sender.CreateBatchAsync();
+
+                batch.TryAdd(new ServiceBusMessage(Encoding.UTF8.GetBytes("This is a message")));
+                batch.TryAdd(new ServiceBusMessage(Encoding.UTF8.GetBytes("This is another message")));
+                batch.TryAdd(new ServiceBusMessage(Encoding.UTF8.GetBytes("So many messages")));
+
+                await sender.SendBatchAsync(batch);
+            }
+        }
+
+        [Test]
+        public async Task SenderCanSendZeroLengthMessageBatch()
```

Comment:
nit: this title made me think we were sending an empty batch rather than a batch with one message with an empty body.